Backendless/Service/MessagingService.cs

```diff
@@ -68,12 +67,87 @@ public MessagingService()
           DeviceId = BitConverter.ToString( (byte[]) deviceId ).Replace( "-", "" ),
           OsVersion = System.Environment.OSVersion.Version.Major.ToString( CultureInfo.InvariantCulture )
         };
-    #elif UNITY
+#elif UNITY
       _deviceRegistrationDto = new DeviceRegistration();
-    #endif
+#endif
     }
 
-  #if UNITY
+#if UNITY_ANDROID || UNITY_IPHONE
```

Comment:
@holailusoria has it been tested in Unity?

Reply:
Not yet